src/plugins/kibana_overview/public/components/_overview.scss

```diff
@@ -0,0 +1,154 @@
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+// Local page variables
+$pageWidth: 1200px;
+
+.kbnOverviewWrapper {
+  background-color: $euiColorEmptyShade;
+  display: flex;
+  flex-direction: column;
+  min-height: calc(100vh - #{$euiHeaderHeightCompensation});
+}
+
+.kbnOverviewHeader {
+  background-color: $euiPageBackgroundColor;
+  border-bottom: $euiBorderWidthThin solid $euiColorLightShade;
+}
+
+.kbnOverviewHeader__inner {
+  margin: 0 auto;
+  max-width: $pageWidth;
+  padding: $euiSizeXL $euiSize;
+}
+
+// Accounting for no `flush="both"` prop on EuiButtonEmpty
+.kbnOverviewHeader__actionItem .euiButtonEmpty__content {
+  padding-left: 0;
+  padding-right: 0;
+}
+
+.kbnOverviewContent {
+  margin: 0 auto;
+  max-width: $pageWidth;
+  padding: $euiSizeXL $euiSize;
+  width: 100%;
+}
+
+// Prevent children from rendering as flex items
+.kbnOverviewGettingStarted__content {
+  display: block;
+}
+
+.kbnOverviewGettingStarted__apps .euiIcon__fillSecondary {
+  fill: $euiColorDarkestShade;
+}
+
+.kbnOverviewApps__item {
+  .kbnOverviewApps__group--primary & {
+    @include euiBreakpoint('m', 'l', 'xl') {
+      max-width: calc(50% - #{$euiSizeM * 2});
+    }
+  }
+
+  .kbnOverviewApps__group--secondary & {
+    @include euiBreakpoint('m', 'l', 'xl') {
+      max-width: calc(25% - #{$euiSizeM * 2});
+    }
+  }
+}
+
+.kbnOverviewNews__content article {
+  & + article {
+    margin-top: $euiSizeL;
+  }
+
+  &,
+  header {
+    & > * + * {
+      margin-top: $euiSizeXS;
+    }
+  }
+
+  h3 {
+    font-weight: inherit;
+  }
+}
+
+.kbnOverviewMore__item {
+  @include euiBreakpoint('m', 'l', 'xl') {
+    max-width: calc(33.333333333333333% - #{$euiSizeM * 2});
+  }
+}
+
+.kbnOverviewSolution__icon {
+  background-color: $euiColorEmptyShade !important;
+  box-shadow: none !important;
+  height: $euiSizeL * 2;
+  padding: $euiSizeM;
+  width: $euiSizeL * 2;
+}
+
+.kbnOverviewSupplements--noNews .kbnOverviewMore {
+  h2 {
+    @include euiBreakpoint('m', 'l', 'xl') {
+      text-align: center;
+    }
+  }
+
+  .kbnOverviewMore__content {
+    @include euiBreakpoint('m', 'l', 'xl') {
+      justify-content: center;
+    }
+  }
+}
+
+.kbnOverviewData--expanded {
+  flex-direction: column;
+
+  &,
+  & > * {
+    margin-bottom: 0 !important;
+    margin-top: 0 !important;
+  }
+}
+
+.kbnOverviewDataAdd__actions {
+  // Prevent children from rendering as flex items
+  display: block;
+
+  // Accounting for no `flush="both"` prop on EuiButtonEmpty
+  .euiButtonEmpty__content {
+    padding-left: 0;
+    padding-right: 0;
+  }
+}
+
+.kbnOverviewDataAdd__content,
+.kbnOverviewDataManage__content {
+  & .euiIcon__fillSecondary {
+    fill: $euiColorDarkestShade;
```

Comment:
I'm guessing you're applying this to an app logo? EuiIcon will allow you to override the overall color to a single one for apps just by passing the `color` prop:
<img width="250" alt="Screen Shot 2020-09-29 at 19 13 01 PM" src="https://user-images.githubusercontent.com/549577/94625918-eb82f200-0287-11eb-8af1-cc341abc5a79.png">

Reply:
In testing this, applying a `color` prop doesn't appear to override the color styles assigned to `.euiIcon__fillSecondary`. The color there remains unchanged.

Reply:
Can you help me debug the EUI side by showing me a screenshot of which icon and where it lives?

Reply:
The issue doesn't appear to be limited to a single icon (though they do all belong to the apps group). Examples shown here include `indexManagementApp`, `securityApp`, and `monitoringApp`, but I'm guessing the problem exists for all of them.

**Icon default appearance:**
![image](https://user-images.githubusercontent.com/3884767/94839086-87c20b80-03e4-11eb-8530-7486fd0cff96.png)

**Attempting to change icon color with `color` prop:**
![image](https://user-images.githubusercontent.com/3884767/94839428-fb641880-03e4-11eb-8dbf-945b6ba5556a.png)

**Results still show `.euiIcon__fillSecondary` in teal color:**
![image](https://user-images.githubusercontent.com/3884767/94839367-e7201b80-03e4-11eb-9bfd-edf6bb82187c.png)

Reply:
Ah I see, yes it looks like the app style icons don't override the secondary fill color when passing a custom color string. As you are showing. But if you use one of our **named** colors, it will. For example, passing `color="text"` should get you the result you are looking for:

<img width="989" alt="Screen Shot 2020-10-01 at 13 06 30 PM" src="https://user-images.githubusercontent.com/549577/94840907-3bc49600-03e7-11eb-8d8a-f70460196690.png">

I'll follow up in EUI with an issue about custom colors. But will probably be low priority.

Reply:
Interesting. Using `color="text"` works, but trying `color="euiColorDarkestShade"` does not.